api/src/certification/enrolment/application/certification-candidate-route.js

```diff
@@ -48,7 +49,17 @@ const register = async function (server) {
                   'date.format': 'CANDIDATE_BIRTHDATE_FORMAT_NOT_VALID',
                 }),
                 'organization-learner-id': Joi.number().empty(null).forbidden(),
+                subscriptions: Joi.array()
+                  .items(
+                    Joi.object({
+                      type: Joi.string().valid(...Object.values(SUBSCRIPTION_TYPES)),
+                      complementaryCertificationId: Joi.number().allow(null),
+                    }),
+                  )
+                  .min(0)
```

Comment:
Nous ne sommes pas obliger de recevoir au moins une souscription ?

```suggestion
.min(1)
```

Reply:
tout à fait, et ça arrive plus tard dans la PR, mais à ce stade le code était encore compliant avec le fait d'envoyer les inscriptions via l'attribut complementaryCertification et laisser subscriptions vide

Reply:
Merci pour la précision, je regarderais l'ensemble du code la prochaine fois :)